src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -80,52 +99,99 @@ def add_profile_args(parser):
         required=False,
         help="The output length (tokens) to use for benchmarking LLMs. (Default: 128)",
     )
-
-
-def add_endpoint_args(parser):
     parser.add_argument(
-        "--url",
+        "--profile-export-file",
         type=str,
-        default="localhost:8001",
         required=False,
-        help="URL of the endpoint to target for benchmarking.",
+        help="Specifies the path that the profile export will be "
+        "generated at. By default, the profile export will not be "
+        "generated.",
     )
     parser.add_argument(
-        "--provider",
+        "--request-rate",
+        type=float,
+        required=False,
+        help="Sets the request rates for load generated by analyzer. ",
+    )
+    parser.add_argument(
+        "--service-kind",
         type=str,
         choices=["triton", "openai"],
+        default="triton",
         required=False,
-        help="Provider format/schema to use for benchmarking.",
+        help="Sets the request rates for load generated by analyzer. "
+        "Describes the kind of service perf_analyzer to "
+        'generate load for. The options are "triton" and '
+        '"openai". The default value is "triton".',
     )
-
-
-def add_dataset_args(parser):
     parser.add_argument(
-        "--dataset",
-        type=str,
-        default="OpenOrca",
-        choices=["OpenOrca", "cnn_dailymail"],
+        "--streaming",
+        action="store_true",
+        required=False,
+        help=f"Enables the use of streaming API. This flag is "
+        "only valid with gRPC protocol. By default, it is set false.",
+    )
+    parser.add_argument(
+        "--sync",
```

Comment:
1. Why have options for both `--sync` and `--async`? What's the default? Sounds like the default will change based on some model characteristics?
2. What does "sequential"  model mean?

Reply:
Sequential is a triton stateful model. It's mentioning is not needed here.

Reply:
How would PA detect that the model is stateful or not? The descriptions make it sound like the default choice of sync or async depends on the model.